tests/unit/amo/pages/TestAddon.js

```diff
@@ -1099,7 +1099,7 @@ describe(__filename, () => {
       getClientCompatibility: getClientCompatibilityFalse,
     });
     expect(root.find(AddonCompatibilityError)).toHaveLength(0);
-    expect(root.find(Button)).toHaveLength(1);
+    expect(root.find(GetFirefoxButton)).toHaveLength(1);
```

Comment:
should we show in one of the tests that it doesn't show up when the browser already is Firefox?

Reply:
Sure, I guess that wasn't covered before, but we can do that.